transport/handler.go

```diff
@@ -37,3 +68,17 @@ type Handler interface {
 		resw ResponseWriter,
 	) error
 }
+
+// OnewayHandler handles a single, transport-level, oneway request.
+type OnewayHandler interface {
+	// Handle the given oneway request
+	//
+	// An error may be returned in case of failures. BadRequestError must be
```

Comment:
There will be no "expected" error types from oneway so no BadRequests will be supported. All errors from oneways are unexpected and we'll let the one way inbound decide how it wants to handle failures.